src/NLog.Targets.Redis/NLog.Targets.Redis.csproj

```diff
@@ -30,12 +30,16 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
     <PackageReference Include="NLog" Version="4.0.0" />
-    <PackageReference Include="StackExchange.Redis.StrongName" Version="1.0.488" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.5' ">
     <PackageReference Include="NLog" Version="4.5.0" />
     <PackageReference Include="StackExchange.Redis.StrongName" Version="1.1.605" />
```

Comment:
We should only have one effective reference here, RE: the above comment.

Reply:
I'll get rid of the conditionals, as the same version can be used in net461 and standard2.0